lib/view/pages/profile/profile_page.dart

```diff
@@ -384,10 +508,14 @@ class _ProfileState extends State<ProfilePage> {
                                           navigateToConnections(context),
                                     ),
                                   ),
-                                  Expanded(
+                                   Expanded(
                                     child: ProfileButton(
-                                        value: '$roundedTotalTime hrs',
-                                        title: 'Time Played'),
+                                      value:
+                                          '$roundedTotalTime hrs',
+                                      title: 'Time Played',
+                                      onPressed: () =>
+                                          navigateToStats(context),
```

Comment:
Thank you for adding this! Could you perhaps add a primary border to the button? just so it looks a bit more like a button?